translations/edx-platform/conf/locale/en/LC_MESSAGES/djangojs.po

```diff
@@ -12079,7 +12079,7 @@ msgstr "Saudi Irrigation Organization"
 msgid "ministry_of_environment_water_and_agriculture"
 msgstr "Ministry of Environment, Water, and Agriculture"
 
-msgid "general_presidency_of_the_affairs_of_al_masjid_al_haram_and_the_prophet_s_mosque"
+msgid "general_presidency_of_the_affairs_of_al_masjid_al_haram_and_the_prophets_mosque"
```

Comment:
@andrey-canon @johanseto why we're editing the strings here, and not in the `custom-translations`?